### ISSUE TITLE
CSS: AgentTicketZoom Article Row with misleading background-color for System E-Mails

### ISSUE THREAD

**Issue by hcamminadi** [RotherOSS/otobo#803]
**Situation:** 
- System E-Mails are configured to be visible for customers
- E-Mail is sent to the ticket system e.g. for an existing ticket
- No custom postmaster filter manipulates Mail Headers
- Article row E-Mail is displayed "red" as if it would not be visible for customers but database value for article.is_visible_for_customer=1 (which is correct with the above configuration)

**Suggestion:**
The reason is that article row is displayed with CSS class "system Outgoing VisibleForCustomer". 
Core.TicketDetail.css has an element for "system" that sets the color to "red" but is missing an element for "VisibleForCustomer" that overwrites the color. 

As a quick fix a custom css file under \thirdparty\ZZZCustom with the following content should help:

```css
.UseArticleColors tr.VisibleForCustomer > td {
    background: #fff !important;
    border-bottom-color: #ccc;
    border-color: #ccc;
}
.UseArticleColors tr.VisibleForCustomer.Active > td,
.UseArticleColors tr.VisibleForCustomer:hover > td {
    background-color: #e5e5e5 !important;
    border-color: #e5e5e5;
    border-right-color: #ccc !important;
}
```

**Comment by svenoe** [comment]
Hi hcamminadi,

thanks for pointing it out - the change to red system articles was not intended. I fixed this, but we will only implement it in 10.1, to not confuse people. You will have to use your fix for a bit, until it is released.

Best regards, Sven

**Comment by bschmalhofer** [comment]
Updated the tags of this issue. I wondered what that was when I looked at the diffs between rel-10_0 and rel-10_1 for #1033.
